### ISSUE TITLE
StORF reporter fails on specific gffs but not others

### ISSUE THREAD

**Issue by LucyDillon** [NickJD/StORF-Reporter#4]
Hi,

I have come across the following issue when running this command (below) on 5k+ gffs:
Command: 

```
StORF-Reporter -anno Bakta  Multiple_GFFs -p ./ -sout True
```

error:
```
Traceback (most recent call last):
  File "/users/40309916/.local/bin/StORF-Reporter", line 8, in <module>
    sys.exit(main())
  File "/users/40309916/.local/lib/python3.10/site-packages/StORF_Reporter/StORF_Reporter.py", line 916, in main
    StORF_Filler(Reporter_options, Reporter_StORFs)
  File "/users/40309916/.local/lib/python3.10/site-packages/StORF_Reporter/StORF_Reporter.py", line 585, in StORF_Filler
    StORFs = find_after_StORFs(Reporter_options, Contig_URS, track_prev_start, track_prev_stop, track_prev_contig)  # Changed to prev stop because we are switching from previous contig
UnboundLocalError: local variable 'track_prev_start' referenced before assignment
```

I have attached the gff which this failed on and if you have any insight of what may be causing the issue please let me know :)
[MGYG000290771.gff3.zip](https://github.com/user-attachments/files/17117562/MGYG000290771.gff3.zip)

**Comment by LucyDillon** [comment]
This is now fixed with the update (v1.4.1) 😁 thank you!

**Comment by NickJD** [comment]
Great. Do let me know if you have any further issues or suggestions.
Thanks for the feedback!

The fix was as follows - 
''As highlighted by @LucyDillon in issue https://github.com/NickJD/StORF-Reporter/issues/4, StORF-Reporter was falling over when presented with an input contig WITHOUT any previous annotation. This was caused by the fact that CDS etc annotations presented to StORF-Reporter are used as anchors in the 'StORF-Filler' function which fits StORFs in the correct order in the new computed GFF file.''

v1.4.1 Release - https://github.com/NickJD/StORF-Reporter/releases/tag/v.1.4.1
